src/main/java/duke/utility/Storage.java

```diff
@@ -0,0 +1,73 @@
+package duke.utility;
+
+import duke.task.Task;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
```

Comment:
Be more consistent with order of import statements. Put ```import java.io```s together and ```import java.util```s together.